src/main/java/org/sonatype/gradle/plugins/scan/nexus/iq/NexusIqScanTask.java

```diff
@@ -131,6 +131,15 @@ public void scan() {
     }
   }
 
+  private Properties buildProperties() {
+    Properties properties = new Properties();
+    if (!StringUtils.isAllBlank(extension.getDirIncludes(), extension.getDirExcludes())) {
```

Comment:
Question. This expects a user must enter **both** the `dirIncludes` and `getExcludes`. They can't just enter one. Is that the intended behaviour?

Reply:
> Question. This expects a user must enter **both** the `dirIncludes` and `dirExcludes`. They can't just enter one. Is that the intended behaviour?

Good catch!
1885f59e0595ffe731249ad188df05b66cfb9697